job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -272,8 +286,10 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
             Collection<SeedReference> seedJobReferences = descriptor.getTemplateJobMap().get(templateName);
             Collection<SeedReference> matching = Collections2.filter(seedJobReferences, new SeedNamePredicate(seedJobName));
             if (!matching.isEmpty()) {
-                seedJobReferences.removeAll(matching);
```

Comment:
I'd like to keep dry-run code as close as possible to the one that would be executed if it was actually running, if it's ok with you.